subsys/bluetooth/Kconfig.iso

```
@@ -0,0 +1,102 @@
+# Bluetooth Isochronous Channel configuration options
+
+# Copyright (c) 2016 Intel Corporation
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_ISO
+	bool # hidden
```

yes, a promptless symbol is hidden, so no need to state that here.

If you want to make a comment, then make it a proper comment explaining why this symbol is hidden and describe for what it is used.

For example in the help text, like here:
https://github.com/zephyrproject-rtos/zephyr/blob/e1481fee4aa1e61ccef8e64e77aa347b6dd10493/modules/mbedtls/Kconfig#L8-L14

But as minimum remove the comment as it provides no value or info to the reader.
```suggestion
	bool
```

---

Done. I just removed comment there. You can easily guess what this option is for, so I didn't add a comment under